### ISSUE TITLE
[FE] 10-10EZ Short-form: Add labels for Edit Dependent button(s) > [Markup and meta data] Visual labels aren't included in the accessible label. (09.05.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va-forms-system-core#358]
#### VFS team name
10-10 Team

#### VFS product name
Health Care Application 10-10EZ

#### VFS feature name
10-10EZ Short Form (Auto-qualifiers)

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Visual labels aren't included in the accessible label.

### Issue Details
On Step 4 of 6 [disability rating <50%]: Household Information, when a user is adding multiple dependents, an Edit button is paired with each dependent. Each of these Edit buttons have identical accessible names, "Edit Dependent." For screen reader users tabbing through the buttons, it's difficult to identify which dependent will be edited by following which button.        

### Link, screenshot or steps to recreate
1. Log into the RI as user 11.
2. Complete the form up to Step 4 of 6.
3. For "Do you have any dependents to report?" answer Yes.
4. Enter information for a dependent.
5. Click "Add another Dependent"
6. Enter information for a dependent.
7. Click ""Add another insurance policy.""
8. Activate a screen reader (recommend NVDA on Windows or VoiceOver on Mac for testing).
9. Press [tab] or [shift] + [tab] to move forward and backward through the interactive elements on the page.
10. Note that the edit buttons for each depdendent are announced only as "Edit Dependent."

### VA.gov Experience Standard
[Category Number 09, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.5.3_A

---

### Platform Recommendation
Each edit button should have a unique accessible name. This can be accomplished either by changing the button text (`<button>Edit [name of dependent]</button>`) or by providing a unique aria-label (`<button aria-label="Edit [name of dependent]">Edit</button>`).

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
Issue moved to [department-of-veterans-affairs/va.gov-team #42856](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42856) via [**ZenHub**](https://www.zenhub.com/)

**Comment by taylorkaren** [comment]
Closing this ticket as it will be completed on ticket department-of-veterans-affairs/va.gov-team #42856.
